packages/graphql/tests/integration/advanced-filtering.int.test.ts

```diff
@@ -680,6 +680,288 @@ describe("Advanced Filtering", () => {
+                            CREATE (:${movieType.name} {title: $matrixRevolutions})
+                        `,
+                    { matrix, matrixReloaded,  matrixRevolutions }
+                );
+
+                const query = `
+                            {
+                                ${movieType.plural}(where: { title_GT: "${matrix}" }) {
+                                    title
+                                }
+                            }
+                        `;
+
+                const gqlResult = await graphql({
+                    schema: await neoSchema.getSchema(),
+                    source: query,
+                    contextValue: neo4j.getContextValuesWithBookmarks(session.lastBookmark()),
+                });
+
+                if (gqlResult.errors) {
+                    console.log(JSON.stringify(gqlResult.errors, null, 2));
+                }
+
+                expect(gqlResult.errors).toBeUndefined();
+
+                expect((gqlResult.data as any)[movieType.plural]).toHaveLength(2);
+                expect((gqlResult.data as any)[movieType.plural]).toEqual([
+                    { title: matrixReloaded },
+                    { title: matrixRevolutions },
+                ]);
+            } finally {
+                await session.close();
+            }
+        });
+
+        test("should find Movies LT string", async () => {
+            const session = await neo4j.getSession();
+
+            const movieType = generateUniqueType("Movie");
+
+            const typeDefs = `
+                        type ${movieType.name} {
+                            title: String
+                        }
+                    `;
+
+            const neoSchema = new Neo4jGraphQL({
+                config: {
+                    features: {
+                        filters: {
+                            String: {
+                                LT: true,
+                                GT: true,
+                                LTE: true,
+                                GTE: true,
+                            },
+                        },
+                    },
+                },
+                typeDefs,
+            });
+
+            const matrix = "The Matrix";
+            const matrixReloaded = "The Matrix Reloaded";
+            const matrixRevolutions = "The Matrix Revolutions";
+
+            try {
+                await session.run(`
+                            CREATE (:${movieType.name} {title: $matrix})
+                            CREATE (:${movieType.name} {title: $matrixReloaded})
+                            CREATE (:${movieType.name} {title: $matrixRevolutions})
+                        `,
+                    { matrix, matrixReloaded,  matrixRevolutions }
+                );
+
+                const query = `
+                            {
+                                ${movieType.plural}(where: { title_LT: "${matrixRevolutions}" }) {
```

Comment:
Same as before, but with a title longer (e.g. "The Englishman who Went up a Hill but Came down a Mountain")